Formula/streamsets-datacollector.rb

```diff
@@ -0,0 +1,59 @@
+class StreamsetsDatacollector < Formula
+  desc "DataOps Platform for Modern Data Movement"
+  homepage "https://streamsets.com"
+  url "https://archives.streamsets.com/datacollector/3.3.0/tarball/streamsets-datacollector-core-3.3.0.tgz"
+  sha256 "c980e4ecca091c8be2d7769d5c0a4a469e8a2096d1f70b1d502536520d6f980f"
+
+  bottle :unneeded
+  depends_on :java => "1.8"
```

Comment:
Can we leave a new line between `bottle :unneeded` and the first `depends_on`? Thanks.